méli-mélo/méli-mélo-en.html

```diff
@@ -0,0 +1,95 @@
+	<li>Must include an implementation plan that explain the steps taken by the sponsored department on how they will actively work the feature to get it integrated into GCWeb.</li>
+	<li>Must no contain any interference with wet-boew and GCWeb.</li>
+	<li>Each individual sub-feature and style must be demoed.</li>
+	<li>Working demo must be published using the GCWeb Jekyll template</li>
+	<li>Project name must be in this format. Year and month of the publication time of the feature followed by a short name identifying the feature</li>
+</ul>
+
+<h2>Contribution instruction</h2>
+
+<ul>
+	<li>Code or/and re-organize the feature by levering GCWeb Jekyll theme</li>
+	<li>Create a project folder in the <code>méli-mélo</code> GCWeb root folder</li>
+	<li>Loading the feature must not change the state of generic page. Feature must be explicitly activated through HTML by the web publisher, like the wet-boew plugin or the usage of a CSS class onto an HTML element.</li>
+	<li>The project name must follow this notation: year-month-projectName. The year and month must correspond to the feature initial publication date. For example "2021-05-steps"</li>
+	<li>The implementation plan should be published along with the project web pages. Each item of the implementation plan must be dated because we are going to use to measure the integration progress into GCWeb. The expectation is to get the méli-mélo feature fully integrated into GCWeb within less than one (1) year.</li>
+	<li>Test your code and submission by using your GitHub pages</li>
+	<li>Submit the méli-mélo feature through a new GitHub PR in the GCWeb project.</li>
+	<li>Once approved by the technical review, it will be assigned to a méli-mélo compilation and released on Canada.ca at the next release windows between 1 to 3 weeks.</li>
+	<li>Update the méli-mélo feature code by executing the implementation plan and addressing each todos identified by the wet-boew technical review team.</li>
+	<li>Participate at the weekly wet-boew Tuesday afternoon code sprint</li>
+	<li>Create the GCWeb provisional/stable feature aligned with our enterprise design approach.</li>
+</ul>
+
+<h2>Technical review checklist</h2>
+
+<ul>
+	<li>Ensure the project sponsor are clearly identified.</li>
+	<li>Project folder name follow the naming convention</li>
+	<li>Each JavaScript feature and each style are demoed</li>
+	<li>Perform a code review to ensure there are no override or conflict with GCWEb or/and wet-boew</li>
+	<li>Quickly check if we don't notice any major or obvious web accessibility and security issue. However, the accessibility conformance remain the responsibility of the publisher when implementing the feature into a web page</li>
+	<li>Ensure the feature don't automatically change the state of a generic page when loaded. It must not impact any content unless when the feature are explicitly activated through the HTML code, like through the use of CSS class or the presence of a data attribute.</li>
```

Comment:
"the feature don't" -> "the feature doesn't" (singular), and "the feature are" => "the feature is".
"unless when" -> "unless"
"use of CSS class" -> "use of a CSS class"